File: Angular9/src/entities/aviocompany.ts
import { Destination } from './destinations';

export class AvioCompan
{
    id : number;
    name : string;
    description : string;
    address: string;
    destinations: Array<Destination>;
    

    constructor(name: string, description: string, destinations: Array<Destination>)
    {
        this.name = name;
        this.description = description;
        this.destinations = destinations;
        destinations = new Array<Destination>();
    }
}

File: WebAPI/WebAPI/Models/AvioCompany.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.Text.Json.Serialization;

namespace WebAPI.Models
{
	public class AvioCompany
	{
		public int AvioCompanyId { get; set; }
		public string Name { get; set; }
		public string Description { get; set; }
		public string Address { get; set; }
		public List<Destination> Destinations { get; set; }
		[JsonIgnore]
		public List<User> Users { get; set; }

		public AvioCompany()
		{

		}
	}
}
